expectise/mocks.py

```diff
@@ -102,5 +102,5 @@ def __set_name__(self, owner: Type, name: str) -> None:
 
 
 def tear_down():
-    """Reset mocking behaviour so that further tests can be run without any interference."""
+    """Reset mocking behavior so that further tests can be run without any interference."""
```

Comment:
what's wrong with US English 😛 ?

Reply:
wahou I didn't know it was written differently in the US.